### PR TITLE
document proxy ssl_disabled_ciphers option

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by domcleal** [comment]
Thanks, looks good. Could you also add a shorter description to the general settings bit in section 4.3.2?  It doesn't need to be detailed, it can refer/link to the later section for detail (e.g. the list of current and known ciphers should be left out).

**Comment by tbrisker** [comment]
@domcleal done

File: _includes/manuals/1.12/4.3.10_smartproxy_ssl.md [hunk 1]
```diff
@@ -29,6 +29,28 @@ Configure the locations to the SSL files in `/etc/foreman-proxy/settings.yml`, p
 #- foreman.dev.domain
 </pre>
 
+##### SSL cipher suites
+
+By default, the smart proxy permits the following SSL cipher suites:
+* ECDHE-RSA-AES128-GCM-SHA256
```
Comment (domcleal) [inline _includes/manuals/1.12/4.3.10_smartproxy_ssl.md:36]: I think you need a line break above here, it's not rendering as a list.

Comment (tbrisker) [inline _includes/manuals/1.12/4.3.10_smartproxy_ssl.md:36]: fixed

**Comment by domcleal** [comment]
Thanks @tbrisker!

**Pull request merged**